integration_tests/pages/page.ts

```diff
@@ -32,13 +32,6 @@ export default abstract class Page {
 
   manageDetails = (): PageElement => cy.get('[data-qa=manageDetails]')
 
-  shouldContainButtonLink(text: string, href: string, buttonLinkElement: JQuery<HTMLElement>): void {
```

Comment:
Would you mind just noting the reason for the removal here (that it's now unused) in the commit message? Then we should be good to go!